export const customHttpException = {
  USER_ALREADY_EXISTS: {
    customCode: 1010,
    message: 'User already exists.',
  },
  INVALID_CREDENTIALS: {
    customCode: 1020,
    message: 'Invalid credentials provided.',
  },
  RESOURCE_NOT_FOUND: {
    customCode: 1030,
    message: 'The requested resource was not found.',
  },
};
